react/__tests__/components/ProductName.test.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import ProductName from './../../ProductName'
+import { render } from 'enzyme'
+import PropTypes from 'prop-types'
+
+describe('<ProductName />', () => {
+  const defaultProps = {
+    loaderClass: '',
+    name: 'ProductTest',
+  }
+
+  const context = { culture: { currency: 'USD' } }
+
+  const renderComponent = customProps => {
+    const props = {
+      ...defaultProps,
+      ...customProps,
+    }
+    return render(<ProductName {...props} />, {
+      context,
+      childContextTypes: {
+        culture: PropTypes.object,
+      },
+    })
+  }
+
+  it('should be mount', () => {
+    expect(renderComponent()).toBeDefined()
+  })
+
+  it('should match snapshot', () => {
```

Comment:
"should match the snapshot"

Reply:
Be more specific on what you're expecting to check in the snapshot.